### PR TITLE
MAUI-5297-Bind JSON data to .NET MAUI DataForm

### PR DESCRIPTION
**Pull request by Rishvanth-Kumaresan**
Hi @MuniappanSubramanian ,

I have prepared a sample for the KB regarding "How to bind JSON data to .NET MAUI DataForm (SfDataForm)". Kindly review it.

Regards,
Rishvanth C K

### CODE REVIEW

File: LoadJSONData/LoadJSONData/LoadJSONData.csproj [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		<MauiFont Include="Resources\Fonts\*" />
+
+		<!-- Raw Assets (also remove the "Resources\Raw" prefix) -->
+		<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="7.0.0" />
+		<PackageReference Include="Syncfusion.Maui.Core" Version="*" />
+		<PackageReference Include="Syncfusion.Maui.DataForm" Version="*" />
```
Comment (MuniappanSubramanian) [inline LoadJSONData/LoadJSONData/LoadJSONData.csproj:53]: Syncfusion.Maui.Core - Need to add?

File: LoadJSONData/LoadJSONData/Sample/Source/ItemSourceProvider.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+namespace LoadJSONData
+{
+    public class ItemSourceProvider : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            if (sourceName == "City")
+            {
+                List<string> city = new List<string>() { "Chennai", "California", "Calabria", "India", "Qatar" };
+                return city;
```
Comment (MuniappanSubramanian) [inline LoadJSONData/LoadJSONData/Sample/Source/ItemSourceProvider.cs:12]: California", "Calabria", "India", "Qatar - Are countries. Update it

Comment (Rishvanth-Kumaresan) [reply LoadJSONData/LoadJSONData/Sample/Source/ItemSourceProvider.cs:12]: Updated the city list.

File: LoadJSONData/LoadJSONData/LoadJSONData.csproj (hunk 1 above)
Comment (Rishvanth-Kumaresan) [reply LoadJSONData/LoadJSONData/LoadJSONData.csproj:53]: Removed Syncfusion.Maui.Core package.

**Review by MuniappanSubramanian — APPROVED** [review]
This is fine

**Pull request merged**